components/ws-manager/pkg/manager/imagespec.go

```diff
@@ -30,16 +30,9 @@ func (m *Manager) GetImageSpec(ctx context.Context, req *regapi.GetImageSpecRequ
 	}
 
 	var (
-		span        opentracing.Span
-		traceID, ok = pod.Annotations[wsk8s.TraceIDAnnotation]
+		span opentracing.Span
 	)
-	if ok {
-		spanCtx := tracing.FromTraceID(traceID)
-		span = opentracing.StartSpan("GetImageSpec", opentracing.FollowsFrom(spanCtx))
-		ctx = opentracing.ContextWithSpan(ctx, span)
-	} else {
-		span, ctx = tracing.FromContext(ctx, "GetImageSpec")
-	}
+	span, ctx = tracing.FromContext(ctx, "GetImageSpec")
```

Comment:
nit:
```suggestion
	span, ctx := tracing.FromContext(ctx, "GetImageSpec")
```

then the `var` block can go away